blocks/callout/callout.css

```diff
@@ -0,0 +1,129 @@
+.callout {
+  > div {
+    display: flex;
+    flex-direction: column;
+    background: var(--theme-clr-background);
+    height: 42.625rem;
+  }
+
+  .callout-image {
+    flex: 1;
+    width: 100%;
+    height: 50%;
+
+    picture, picture img {
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
+  }
+
+  .callout-content {
+    height: 50%;
+    color: var(--theme-clr-text);
+    padding: 5rem .9375rem;
+
+    h3 {
+      font-family: MiloPro-Bold, Helvetica, sans-serif;
+      font-size: var(--heading-font-size-m);
+      letter-spacing: -.03125rem;
+      margin: 0 0 1rem;
+    }
+
+    p {
+      font-family: OpenSans, Helvetica, sans-serif;
+      font-size: 1rem;
+      line-height: 1.5rem;
+      letter-spacing: .015625rem;
+    }
+
+    .button-container {
+      margin: 0;
+      > a {
+        display: inline-block;
+        margin-top: 1.5rem;
+        min-width: 15.625rem;
+        padding: .875rem;
+        background: 0 0;
+        border: 2px solid #fff;
+        border-radius: .125rem;
+        font-family: MiloPro-Bold, Helvetica, sans-serif;
+        font-size: 1rem;
+        text-align: center;
+        line-height: 1.5rem;
+        letter-spacing: .03125rem;
+        color: #fff;
+      }
+    }
+  }
+}
+
+.callout-wrapper {
+  margin-left: auto;
+  margin-right: auto;
+  padding: 0 !important;
+  margin-bottom: 3.125rem !important;
+}
+
+@media (min-width: 67.5625rem) {
+
+  .callout {
+    > div {
+      flex-direction: row;
+      gap: 0;
+      height: 35rem;
+    }
+    .callout-image {
+      width: 50%;
+      height: 100% !important;
+    }
+    
+  }
+  .callout-content {
+    width: 50%;
+    height: 100%;
+    align-self: center;
+    padding: 0 2.5rem !important;
+    margin: auto;
+    h3 {
+      font-size: var(--heading-font-size-xl) !important;
+      max-width: 26.3125rem !important;
+    }
+    p {
+      max-width: 26.3125rem !important;
+    }
```

Comment:
Are all the !important statements necessary? If the selectors are scoped properly they should override any global properties.  While they can be necessary on occasion, over use can make things tough, when you 'really' need to override something. A better approach is to make your selectors specific.

Reply:
Not sure if they're all needed at this point, to be fair!
Thank you for the hint, I will review the selectors scope!